addons/aws-elb-controller/enable

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+set -e
+
+source $SNAP/actions/common/utils.sh
+CURRENT_DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+source $CURRENT_DIR/../common/utils.sh
+
+usage() { echo "Usage: sudo microk8s enable aws-elb-controller [-c <cluster_name>]" 1>&2; exit 1; }
+
+while getopts ":c:" o; do
+    case "${o}" in
+        c)
+            CLNAME=${OPTARG}
+            ;;
+        *)
+            usage
+            ;;
+    esac
+done
+shift $((OPTIND-1))
+
+if [ -z "${CLNAME}" ]; then
+    usage
+fi
+
+"$SNAP/microk8s-enable.wrapper" core/helm3
+"$SNAP/microk8s-enable.wrapper" core/dns
+
+KUBECTL="$SNAP/microk8s-kubectl.wrapper"
+HELM="${SNAP}/microk8s-helm3.wrapper"
+
+if is_ec2_instance
+then
+  INSTANCE_ID=$(curl -sS http://169.254.169.254/latest/meta-data/instance-id)
```

Comment:
I would add an `echo ...` here explaining what is happening, just to ensure that the user is aware of the underlying actions.